src/Resolvers/Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver/SdkResolvers/Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver/Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver.xml

```diff
@@ -1,3 +1,4 @@
 <SdkResolver>
 <Path>..\..\Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver.dll</Path>
+<ResolvableSdkPattern>Microsoft\.NET\.SDK\.WorkloadAutoImportPropsLocator|Microsoft\.NET\.SDK\.WorkloadManifestTargetsLocator|Microsoft\..*|Samsung\..*|GtkSharp\..*</ResolvableSdkPattern>
```

Comment:
```suggestion
<ResolvableSdkPattern>Microsoft\..*|Samsung\..*|GtkSharp\..*</ResolvableSdkPattern>
```

So it looks like anything starting with `Microsoft.`, `Samsung.`, or `GtkSharp.` will match.  Given that, I don't think we need to special case the WorkloadAutoImportPropsLocator or WorkloadManifestTargetsLocator since they both start with Microsoft.

Did you look at all the packs from all the manifests (including Tizen) to make sure that they all match these patterns?

Reply:
Good call, I can remove the specific matches. These broad patterns are sufficient to cover all the packs I could find on github - so there _is_ a risk that there's some workload that isn't OSS and is broadly used that these patterns wouldn't cover.